composer.json

```diff
@@ -43,6 +42,7 @@
     "require-dev": {
         "doctrine/coding-standard": "^12.0.0",
         "mikey179/vfsstream": "^1.6.11",
+        "monolog/monolog": "^3.5.0",
```

Comment:
Please add `"monolog/monolog"` also in the `suggest` section with an comment, that this is required for the console commands.